content/en/install.md

```diff
@@ -76,7 +76,7 @@ On all of Windows, macOS, and Linux:
 
 ### Advanced users
 
-#### Windows or macOS
+#### Conda
 
 - Install [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
```

Comment:
@mattip can you change this to Miniforge then? And the drop the bit below about `defaults`? We can and should recommend `conda-forge` without any disclaimer now (and that was the point of the issue you're addressing)

Reply:
Done for English. Is there someone to ping for the translations?

Reply:
No, they show up (with notifications) automatically in Crowdin if you don't touch them. This is just a name, though, so why not just make the change in the translations?

Reply:
> And the drop the bit below about defaults?

This is the part that requires help with translation

Reply:
Okay then let's leave that as is. It'll show up in Crowdin.